gotocompany/assets/v1beta2/asset.proto

```diff
@@ -61,4 +61,8 @@ message Asset {
   // The timestamp when the asset was last modified.
   // This information is expected to be maintained by the system.
   google.protobuf.Timestamp update_time = 102;
+
+  // The timestamp when the asset was last refreshed.
+  // This information is expected to be maintained by the system.
+  google.protobuf.Timestamp refreshed_at = 103;
```

Comment:
Is refreshed_at required? Can't it we handle through `update_time` ?

Reply:
Yap, It's capable of handling situations where an asset doesn't receive any single updates, although it already synced/refreshed (should **not** categorize as a stale asset), which is something we cannot detect with `updated_at` field. That's why we introduce that flag